uReplicator-Common/src/main/java/com/uber/stream/kafka/mirrormaker/common/core/KafkaBrokerTopicObserver.java

```diff
@@ -49,24 +47,29 @@
       LoggerFactory.getLogger(KafkaBrokerTopicObserver.class);
 
   private static String KAFKA_TOPICS_PATH = "/brokers/topics";
+  private static String KAFKA_INNER_TOPIC = "__consumer_offsets";
```

Comment:
Rename to KAFKA_INTERNAL_TOPICS and make it a ImmutableList.
Because in later Kafka version, there will be more internal topics, like `__transaction_state`, so make it a list can be extensible in the future.

Reply:
nice suggestion.
already done